scripts/lib/CIME/XML/env_batch.py

```diff
@@ -43,7 +43,7 @@ def set_value(self, item, value, subgroup=None, ignore_type=False):
                 elif value.count(":") == 2:
                     t_spec = "%H:%M:%S"
                 else:
-                    expect(False, "could not interpret format for wallclock time %s"%value)
+                    expect(False, "could not interpret format for wallclock time {}".format(value))
```

Comment:
It is for python3 compatibility. Yes, we aren't using python3, but it is an easy change that doesn't hurt things and will make any future switch to python3 that much easier.